ceph/ceph_admin/osd.py

```diff
@@ -267,3 +267,20 @@ def osd_in(self, config: dict):
             if osd_ids is not None:
                 base_cmd.extend([str(ele) for ele in osd_ids])
         return self.shell(args=base_cmd)
+
+    def flag(self, config: dict):
+        """
+        Execute the command ceph osd set/unset flag.
+        Args:
+            config (Dict): OSD flag configuration parameters
+                command: set/unset
+                base_cmd_args:
+                    verbose: true
+                flag: value of flag
+        Returns:
+          output, error  returned by the command
+        """
+        command = config.get("command")
+        flag = config.get("flag")
+        cmd = f"ceph osd {command} {flag}"
+        return self.shell(args=[cmd])
```

Comment:
Suggestion: Instead of user processing the o/p by the calling function, we can perform those checks in this method itself and return true/false based on status! Would avoid code repetition in future.

Reply:
Yes agreed, however this function is being called by `configure_osd_flag` in [ceph/rados/utils.py](https://github.com/red-hat-storage/cephci/pull/2337/files#diff-5232d7b6ad4cdb18e68353cedf6d3198168562d59e87e2e6882b99ffffc7be1b) 
I have followed the workflow which is already in place for OSD related commands.